src/AspNetCore/Src/serverprocess.cxx

```diff
@@ -832,7 +832,7 @@ SERVER_PROCESS::PostStartCheck(
             pStruErrorMessage->SafeSnwprintf(
                 ASPNETCORE_EVENT_PROCESS_START_NOTREADY_ERROR_MSG,
                 m_struAppFullPath.QueryStr(),
-                m_pszRootApplicationPath,
+                m_pszRootApplicationPath.QueryStr(),
```

Comment:
@pan-wang Would you confirm if this change is required or not?

Reply:
For me, these were required for the upgrade from VS 2015 to VS 2017.

Reply:
Please rename m_pszRootApplicationPath to m_struRootApplicationPath as the type got changed and to follow the naming guideline.